dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityOperationParamsMapperTest.java

```diff
@@ -521,36 +524,6 @@ void shouldFailWhenGlobalSearchAndNoAttributeSpecified()
         exception.getMessage());
   }
 
-  @Test
-  void shouldFailWhenGlobalSearchAndMaxTeLimitReached()
```

Comment:
Why was this removed? To my naked eye, this "kinda" looks correct or am i missing something?

Reply:
because I'm testing everything related to the limits (program, TET and system) in `TrackedEntityMaxLimitServiceTest`
This particular test is now [here](https://github.com/dhis2/dhis2-core/pull/20160/files#diff-7ddf1e75d016bdef7a4b29ca6b8ac85c6e9f749aba027d335a24b5261db3aa6aR105) and [here](https://github.com/dhis2/dhis2-core/pull/20160/files#diff-7ddf1e75d016bdef7a4b29ca6b8ac85c6e9f749aba027d335a24b5261db3aa6aR127)